docs/table-of-contents.md

```diff
@@ -4,7 +4,7 @@ description: The Table of Contents is the left-hand navigation for your site, it
 thumbnail: thumbnails/table-of-contents.png
 ---
 
```

Comment:
@choldgraf the other thing i'm trying to figure out that is confusing related to TOC / NAV
if you go to this page - notice there is no toctree on some pages and sidebar toctrees on others. 
https://mystmd.org/overview/ecosystem 

in one of the repos that i saw, there was a _toc.yml file that in the docs said was deprecated.

can you add info on (maybe i missed this) how you control what's on the left hand side bar for a folder / section ? if it's in the docs i just had a hard time finding it. if this is out of scope for this pr that is fine too - i know this is a nerd snipe situation which is not my goal! 🙃

Reply:
The mystmd.org docs are actually not a great example of this because I think the content of those repositories is actually hosted in different places. It is confusing I agree. In my opinion it would be better for us to `embed` that content and try to accomplish the myst docs with a single repository without stitching together a bunch of repos in a hacky way.

I opened up this issue which I think would help your situation:

- https://github.com/jupyter-book/mystmd/issues/1450